addons/test_mail/tests/test_mail_activity.py

```diff
@@ -759,3 +759,24 @@ def test_week_grouping(self):
         self.assertEqual(groups[0][groupby], pg_groups["overdue"])
         self.assertEqual(groups[1][groupby], pg_groups["today"])
         self.assertEqual(groups[2][groupby], pg_groups["planned"])
+
+@tests.tagged('mail_activity')
+class TestActivitySystray(TestActivityCommon):
+    """Test for systray_get_activities"""
+
+    def test_systray_activities_for_archived_records(self):
+        """
+            Checks if the activities made on the archived records is shown in the
+            systray activities.
+        """
```

Comment:
Typo (is -> are), suggesting
```suggestion
        """ Check that activities made on archived records are shown in the
        systray activities. """
```